config/schema.sql

```diff
@@ -63,6 +47,7 @@ CREATE TYPE enum_activities_type AS ENUM (
     'comment',
     'get_comment',
     'get_asset_comment_reply',
+    'pin_asset',
```

Comment:
The proof is in the pudding: running the reference schema against actual code revealed that we'd missed this value.